apps/client/src/common/api/apiConstants.ts

```diff
@@ -15,8 +13,8 @@ export const RUNTIME = ['runtimeStore'];
 const location = window.location;
 const socketProtocol = location.protocol === 'https:' ? 'wss' : 'ws';
 
-export const serverPort = import.meta.env.DEV ? STATIC_PORT : location.port;
-export const serverURL = import.meta.env.DEV ? `http://${location.hostname}:${serverPort}` : location.origin;
+export const serverPort = location.port;
```

Comment:
When developing the app we spin react in dev mode (usually port 4001). The previous logic achieved that we can still connect to the server at whatever port but have react coming from another port

HMR is a must for application development, can you think of a way we can keep this?
worst case scenario the previous logic should still work
`export const serverPort = import.meta.env.DEV ? STATIC_PORT : location.port`

the developer would need to manually change the STATIC_PORT in case they want to test on a different port
What do you think?

Reply:
Makes sense, wasn't aware of this. Reverted the change. It now works in DEV mode too.